docs/activity.md

```diff
@@ -0,0 +1,51 @@
+# Activity API
+
+The Activity API is an abstraction built on top of the Datafeed's [_Real Time Events_](https://developers.symphony.com/restapi/docs/real-time-events).
+An Activity is basically a user interaction triggered from the chat.
+
+At the moment, two different kinds of activities are supported by the BDK:
+- **Command Activity**: triggered when a message is sent in an `IM`, `MIM` or `Chatroom`
+- **Form Activity**: triggered when a user replies to an [_Elements_](https://developers.symphony.com/symphony-developer/docs/overview-of-symphony-elements) form message
+
+Using the Activity API will help you to make your bot interactions easier and faster to implement.
+
+## Command Activity
+A command activity is triggered when a message is sent in an `IM`, `MIM` or `Chatroom`. 
+
+### How to create a Command Activity
+
+```java
+public class CommandActivityExample {
+
+  public static void main(String[] args) {
+    // TODO
```

Comment:
Do we want to merge even if documentation is not complete ?

Reply:
No no, I need to finish doc before merging